app/src/main/res/layouts/loginActivity/values/input_template_style.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="input_template_style">
+        <item name="android:layout_width">336dp</item>
```

Comment:
Instead of fixed width, use fixed left/right margin to fit different size of screen better.